specification/gp-connect-user-permissions.yaml

```diff
@@ -184,8 +184,8 @@ paths:
         Get a patient's permissions to access the medical record and services
         they can access at their GP practice.
         Returns a `UserPermission` object which includes a list of services and
-        a list of medical record permission sets with a date range for when
-        they are or have been applicable.
+        the `current` and `historical` level of permission has along with a
```

Comment:
Is it 'level of permission **the user** has....'?

Reply:
👀 I've updated it.